.github/workflows/unit-tests.yml

```diff
@@ -1,24 +1,38 @@
-name: TypeScript Unit Tests
-on:
-  pull_request:
-    types:
-      - opened
-      - reopened
-      - ready_for_review
-      - synchronize
+name: Unit tests
+
+on: workflow_call
+permissions: {}
+
+concurrency:
+  group: unit-tests-${{ github.workflow }}-${{ github.head_ref || github.ref_name }}
```

Comment:
Is there any GHA documentation on how this concurrency / group block works?

Reply:
https://docs.github.com/en/actions/using-jobs/using-concurrency